### PR TITLE
updating doc with links for licenses

### PR DESCRIPTION
**Pull request by Moose0621**
The experiance for new users is not very obvious as to `where` they will receive their license. I've included links for Download and Install to help the user find the information.  I will be adding another commit when I have fleshed out the various ways that users get the GHES license.

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks for opening a PR @Moose0621! Let me know when you're ready for a writer review ✨

**Comment by Moose0621** [comment]
> Thanks for opening a PR @Moose0621! Let me know when you're ready for a writer review ✨

@janiceilene I'm ready and could use some help, I updated the PR Comments as to the Why and What changed and looking forward to feedback around this!

**Comment by RaFyyZ8943** [comment]
Pull req is tatics

**Comment by janiceilene** [comment]
It looks like our new action inadvertently closed this :upside_down_face: Sorry about that! Reopening and triaging this for review :yellow_heart:

File: content/admin/overview/managing-your-github-enterprise-license.md [hunk 1]
```diff
@@ -18,13 +18,16 @@ versions:
 
 ### About {% data variables.product.prodname_enterprise %} licenses
 
-When you purchase or renew {% data variables.product.prodname_enterprise %}, you receive a license file to validate your application. A license file has an expiration date and controls the number of user licenses you can add to {% data variables.product.prodname_enterprise %}. After you download and install {% data variables.product.prodname_enterprise %}, uploading the license file unlocks the application for you to use.
+When you purchase or renew {% data variables.product.prodname_enterprise %}, you receive a license file to validate your application. A license file has an expiration date and controls the number of user licenses you can add to {% data variables.product.prodname_enterprise %}. After you [download](https://enterprise.github.com/releases/) and [install](https://docs.github.com/en/enterprise-server/admin/installation/setting-up-a-github-enterprise-server-instance) {% data variables.product.prodname_enterprise %}, uploading the license file unlocks the application for you to use.
```
Comment (mchammer01) [inline content/admin/overview/managing-your-github-enterprise-license.md:21]:
```suggestion
When you purchase or renew {% data variables.product.prodname_enterprise %}, you receive a license file to validate your application. A license file has an expiration date and controls the number of user licenses you can add to {% data variables.product.prodname_enterprise %}. After you download and install {% data variables.product.prodname_enterprise %}, uploading the license file unlocks the application for you to use. For more information about downloading {% data variables.product.prodname_enterprise %}, see the [{% data variables.product.prodname_enterprise %} Releases](https://enterprise.github.com/releases/) website. For information about setting up {% data variables.product.prodname_enterprise %}, see "[Setting up a {% data variables.product.prodname_enterprise %} instance](https://docs.github.com/en/enterprise-server/admin/installation/setting-up-a-github-enterprise-server-instance)."

```

File: content/admin/overview/managing-your-github-enterprise-license.md [hunk 2]
```diff
@@ -18,13 +18,16 @@ versions:
...
 
 If your {% data variables.product.prodname_ghe_server %} license expires, you won't be able to access {% data variables.product.product_location_enterprise %} via a web browser or Git. If needed, you will be able to use command-line utilities to back up all your data. For more information, see "[Configuring backups on your appliance](/enterprise/admin/guides/installation/configuring-backups-on-your-appliance)." If you have any questions about renewing your license, contact {% data variables.contact.contact_enterprise_sales %}.
 
-### Uploading a new license to {% data variables.product.prodname_ghe_server %}
+You can download your {% data variables.product.prodname_ghe_server %} license from your [Enterprise Account](https://docs.github.com/en/enterprise-server/admin/overview/managing-your-github-enterprise-license#uploading-a-new-license-to-github-enterprise-server), if you do not have an Enterprise Account you can login to our [Enterprise Portal](https://enterprise.github.com/download) and download your license.
```
Comment (mchammer01) [inline content/admin/overview/managing-your-github-enterprise-license.md:27]:
```suggestion
You can download your {% data variables.product.prodname_ghe_server %} license from your [Enterprise Account](https://docs.github.com/en/enterprise-server/admin/overview/managing-your-github-enterprise-license#uploading-a-new-license-to-github-enterprise-server), if you do not have an Enterprise Account you can log in to our [Enterprise Portal](https://enterprise.github.com/download) and download your license. 
```

File: content/admin/overview/managing-your-github-enterprise-license.md [hunk 3]
```diff
@@ -18,13 +18,17 @@ versions:
...
+When you purchase or renew {% data variables.product.prodname_enterprise %}, you receive a license file to validate your application. A license file has an expiration date and controls the number of user licenses you can add to {% data variables.product.prodname_enterprise %}. After you download and install {% data variables.product.prodname_enterprise %}, uploading the license file unlocks the application for you to use. For more information about downloading {% data variables.product.prodname_enterprise %}, see the [{% data variables.product.prodname_enterprise %} Releases](https://enterprise.github.com/releases/) website. For information about setting up {% data variables.product.prodname_enterprise %}, see "[Setting up a {% data variables.product.prodname_enterprise %} instance](https://docs.github.com/en/enterprise-server/admin/installation/setting-up-a-github-enterprise-server-instance)."
```
Comment (mchammer01) [inline content/admin/overview/managing-your-github-enterprise-license.md:21]:
```suggestion
When you purchase or renew {% data variables.product.prodname_enterprise %}, you receive a license file to validate your application. A license file has an expiration date and controls the number of user licenses you can add to {% data variables.product.prodname_enterprise %}. After you download and install {% data variables.product.prodname_enterprise %}, uploading the license file unlocks the application for you to use. For more information about downloading {% data variables.product.prodname_enterprise %}, see the [{% data variables.product.prodname_enterprise %} Releases](https://enterprise.github.com/releases/) website. For information about setting up {% data variables.product_location_enterprise %}, see "[Setting up a {% data variables.product.prodname_enterprise %} instance](https://docs.github.com/en/enterprise-server/admin/installation/setting-up-a-github-enterprise-server-instance)."
```

File: content/admin/overview/managing-your-github-enterprise-license.md [hunk 4]
```diff
@@ -18,13 +18,17 @@ versions:
...
+When you purchase or renew {% data variables.product.prodname_enterprise %}, you receive a license file to validate your application. A license file has an expiration date and controls the number of user licenses you can add to {% data variables.product.prodname_enterprise %}. After you download and install {% data variables.product.prodname_enterprise %}, uploading the license file unlocks the application for you to use. For more information about downloading {% data variables.product.prodname_enterprise %}, see the [{% data variables.product.prodname_enterprise %} Releases](https://enterprise.github.com/releases/) website. For information about setting up {% data variables.product_location_enterprise %}, see "[Setting up a {% data variables.product.prodname_enterprise %} instance](https://docs.github.com/en/enterprise-server/admin/installation/setting-up-a-github-enterprise-server-instance)."
```
Comment (mchammer01) [inline content/admin/overview/managing-your-github-enterprise-license.md:21]:
```suggestion
When you purchase or renew {% data variables.product.prodname_enterprise %}, you receive a license file to validate your application. A license file has an expiration date and controls the number of user licenses you can add to {% data variables.product.prodname_enterprise %}. After you download and install {% data variables.product.prodname_enterprise %}, uploading the license file unlocks the application for you to use. For more information about downloading {% data variables.product.prodname_enterprise %}, see the [{% data variables.product.prodname_enterprise %} Releases](https://enterprise.github.com/releases/) website. For information about setting up {% data variables.product.product_location %}, see "[Setting up a {% data variables.product.prodname_enterprise %} instance](https://docs.github.com/en/enterprise-server/admin/installation/setting-up-a-github-enterprise-server-instance)."
```

**Comment by mchammer01** [comment]
Thanks for raising this PR @Moose0621 - I've made a few small tweaks, and will merge this once the checks have passed ✨

File: content/admin/overview/managing-your-github-enterprise-license.md [hunk 5]
```diff
@@ -18,13 +18,17 @@ versions:
...
+When you purchase or renew {% data variables.product.prodname_enterprise %}, you receive a license file to validate your application. A license file has an expiration date and controls the number of user licenses you can add to {% data variables.product.prodname_enterprise %}. After you download and install {% data variables.product.prodname_enterprise %}, uploading the license file unlocks the application for you to use. For more information about downloading {% data variables.product.prodname_enterprise %}, see the [{% data variables.product.prodname_enterprise %} Releases](https://enterprise.github.com/releases/) website. For information about setting up {% data variables.product.product_location %}, see "[Setting up a {% data variables.product.prodname_enterprise %} instance](https://docs.github.com/en/enterprise-server/admin/installation/setting-up-a-github-enterprise-server-instance)."
```
Comment (megbird) [inline content/admin/overview/managing-your-github-enterprise-license.md:21]:
```suggestion
When you purchase or renew {% data variables.product.prodname_enterprise %}, you receive a license file to validate your application. A license file has an expiration date and controls the number of user licenses you can add to {% data variables.product.prodname_enterprise %}. After you download and install {% data variables.product.prodname_enterprise %}, uploading the license file unlocks the application for you to use. For more information about downloading {% data variables.product.prodname_enterprise %}, see the [{% data variables.product.prodname_enterprise %} Releases](https://enterprise.github.com/releases/) website. For information about setting up {% data variables.product.product_location %}, see "[Setting up a {% data variables.product.prodname_enterprise %} instance](/admin/installation/setting-up-a-github-enterprise-server-instance)."
```

File: content/admin/overview/managing-your-github-enterprise-license.md [hunk 6]
```diff
@@ -18,13 +18,17 @@ versions:
...
 
 If your {% data variables.product.prodname_ghe_server %} license expires, you won't be able to access {% data variables.product.product_location_enterprise %} via a web browser or Git. If needed, you will be able to use command-line utilities to back up all your data. For more information, see "[Configuring backups on your appliance](/enterprise/admin/guides/installation/configuring-backups-on-your-appliance)." If you have any questions about renewing your license, contact {% data variables.contact.contact_enterprise_sales %}.
 
-### Uploading a new license to {% data variables.product.prodname_ghe_server %}
+You can download your {% data variables.product.prodname_ghe_server %} license from your [Enterprise Account](https://docs.github.com/en/enterprise-server/admin/overview/managing-your-github-enterprise-license#uploading-a-new-license-to-github-enterprise-server), if you do not have an Enterprise Account you can log in to our [Enterprise Portal](https://enterprise.github.com/download) and download your license.
```
Comment (megbird) [inline content/admin/overview/managing-your-github-enterprise-license.md:28]:
```suggestion
You can download your {% data variables.product.prodname_ghe_server %} license from your [enterprise account](https://enterprise.github.com/download). For more information, see "[Managing your {% data variables.product.prodname_enterprise %} license](/admin/overview/managing-your-github-enterprise-license#uploading-a-new-license-to-github-enterprise-server)." 
```

**Review by megbird — APPROVED** [review]
Did a quick fix and merging now ⚡  Thank you @Moose0621 for opening this up 😄

**Pull request merged**